examples/shader/Shader.cpp

```diff
@@ -151,7 +147,7 @@ class StormBlink : public Effect
         }
 
         // Load the shader
-        return m_shader.loadFromFile("resources/storm.vert", "resources/blink.frag");
+        return true;
```

Comment:
This seems a bit weird... At the very least, the comment needs to be changed. It also seems that `onLoad` doesn't have much of a purpose now, as the `load` function is as follows:

    void load()
    {
        m_isLoaded = sf::Shader::isAvailable() && onLoad();
    }
    
I'm not sure what the best course of action is here, maybe add a TODO in this PR and we can have a new one to revamp the example with a more appropriate design?

Reply:
> At the very least, the comment needs to be changed.

Fixed

Other `onLoad` functions have valid reasons for potentially failing. This particular function won't fail but others might so we can't yet change the signature of `onLoad`.